torch/utils/cpp_extension.py

```diff
@@ -723,12 +723,15 @@ def spawn(cmd):
                             cflags = ['-Xcompiler', flag] + cflags
                         for ignore_warning in MSVC_IGNORE_CUDAFE_WARNINGS:
                             cflags = ['-Xcudafe', '--diag_suppress=' + ignore_warning] + cflags
+                        append_std17_if_no_std_present(cflags)
```

Comment:
Move this to a separate C++17

Reply:
This is, sapling just stacks differently from ghstack, it only hides the parent commit in ReviewStack. Sorry :(